.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-26 16:04:23 -0700 using RuboCop version 0.34.2.
+# on 2015-10-28 13:42:33 -0700 using RuboCop version 0.34.2.
 # The point is for the user to remove these configuration records
```

Comment:
At a glance, I don't see this PR adding any new violations, just continuing existing bad practice, right?